js/common/modules/@arangodb/util.js

```diff
@@ -28,8 +28,8 @@
 
 const _ = require('lodash');
 const fs = require('fs');
-const Chalk = require('chalk').constructor;
-const chalk = new Chalk({enabled: true});
+const Chalk = require('chalk').Instance;
+const chalk = new Chalk({level: 2});
```

Comment:
This _is_ a breaking change, but chalk is only for command line output and shouldn't be part of our publicly bundled modules so this shouldn't affect users.

Reply:
At least not listed here 👍 
- https://www.arangodb.com/docs/3.7/appendix-java-script-modules.html#bundled-npm-modules

Reply:
In the future, we should separate between internal used modules (which must not be pre-installed / pre-bundled) and officially supported ones.

Reply:
I discussed this with @joerg84 and am of the opinion we should stop exposing our bundled modules in ArangoDB 4.